lib/daru/view/adapters/googlecharts.rb

```diff
@@ -184,14 +184,22 @@ def export_html_file(plot, path='./plot.html')
           File.write(path, str)
         end
 
-        def export(_plot, _export_type='png', _file_name='chart')
-          raise 'Not implemented yet'
-        end
-
         def show_in_iruby(plot)
           plot.show_in_iruby
         end
 
+        # Exporting a googlchart to pdf is not working in IRuby notebook because
+        #   the dependency of jspdf is not properly loaded in IRuby notebook.
+        #   TODO: Need to find some other way to export the chart to pdf in
+        #     IRuby notebook.
+        #
+        # @see #Daru::View::Plot.export
+        def export(plot, export_type='png', file_name='chart')
```

Comment:
What if user enters the export_type which is not going to work? Where is the exception for that ?